src/devices/Mcp23xxx/Mcp23x0x.cs

```diff
@@ -2,15 +2,17 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Device.Gpio;
+
 namespace Iot.Device.Mcp23xxx
 {
     /// <summary>
     /// Wraps 8 bit MCP I/O extenders
```

Comment:
NIT: Should be "8-bit".  Should extenders be expanders?

Reply:
Changing.